app/services/export_service_exporters/response_exporter.rb

```diff
@@ -33,7 +33,8 @@ def export_files(response, to:)
 
     def export_file(response:, to:, ext:, type:)
       stored_file = response_export_service.new(response).send(:"export_#{type}")
-      fn = "#{response.reference}_ET3_.#{ext}"
+      company_name_underscored = response.respondent.name.split(/\W/).join('_')
```

Comment:
I think parameterize('_') would do the same thing.

Reply:
thanks for that - have now changed it - should have thought that active support would have an answer :-)